### PR TITLE
LT 1445 ultimate end user docs

### PR DESCRIPTION
**Pull request by BenAndrew-Kainos**

### CODE REVIEW

File: apply_for_a_licence/views/common.py [hunk 1]
```diff
@@ -63,6 +64,7 @@ def get(self, request, **kwargs):
         goodstypes, status_code = get_draft_goods_type(request, draft_id)
         external_locations, status_code = get_external_locations_on_draft(request, draft_id)
         ultimate_end_users, status_code = get_ultimate_end_users(request, draft_id)
+        # TODO: Discuss with Andy why this isn't another endpoint call
```
Comment (Mark-Spiers-kainos) [inline apply_for_a_licence/views/common.py]: have you done?

File: goods/views.py [hunk 2]
```diff
@@ -18,6 +18,7 @@
 from goods.services import get_goods, post_goods, get_good, update_good, delete_good, get_good_documents, get_good_document, delete_good_document, post_good_documents, raise_clc_query
 
 
+
```
Comment (Mark-Spiers-kainos) [inline goods/views.py]: please make 2 lines for pep8 purposes

File: apply_for_a_licence/views/common.py (hunk 1 above)
Comment (BenAndrew-Kainos) [reply apply_for_a_licence/views/common.py]: yeah

File: goods/views.py (hunk 2 above)
Comment (BenAndrew-Kainos) [reply goods/views.py]: done

File: ui_automation_tests/features/submit_standard_application.feature [hunk 3]
```diff
@@ -197,14 +164,25 @@ Feature: I want to indicate the standard licence I want
...
+    Given I go to exporter homepage and choose Test Org
+    When I create a standard application
+    And I click on ultimate end users
+    And I click on ultimate end users add button
+    And I add an end user of type: "government", name: "Mr Smith", website: "https://www.smith.com", address: "London" and country "Ukraine"
+    And I click back link
+    Then "Attach" link is present
+    When I click on attach a document
+    And I upload a file "file_for_doc_upload_test_1.txt"
+    Then "Download" link is present
```
Comment (phlptt) [inline ui_automation_tests/features/submit_standard_application.feature]: is this going to be reliable with not knowing how long itll take for the upload to happen?

File: ui_automation_tests/step_defs/test_standard_application.py [hunk 4]
```diff
@@ -185,3 +185,29 @@ def i_wait_for_end_user_document_to_be_processed(driver):
...
+@then(parsers.parse('"{button}" link is present'))
+def download_and_delete_is_links_are_present(driver, button):
+    shared = Shared(driver)
+    latest_ueu_links = [link.text for link in shared.get_links_of_table_row(-1)]
+    assert button in latest_ueu_links
+
+
+@when("I click on attach a document")
+def click_attach_a_document(driver):
+    UltimateEndUsersListPage(driver).click_on_attach_ultimate_end_user_document(-1)
```
Comment (phlptt) [inline ui_automation_tests/step_defs/test_standard_application.py]: what's the -1 indicate here?

File: ui_automation_tests/step_defs/test_standard_application.py [hunk 5]
```diff
@@ -185,3 +185,29 @@ def i_wait_for_end_user_document_to_be_processed(driver):
         time_no += function_retry_interval
         driver.refresh()
     return False
+
+
+@then(parsers.parse('"{button}" link is present'))
+def download_and_delete_is_links_are_present(driver, button):
+    shared = Shared(driver)
+    latest_ueu_links = [link.text for link in shared.get_links_of_table_row(-1)]
```
Comment (phlptt) [inline ui_automation_tests/step_defs/test_standard_application.py]: what's the -1 indicate here?

File: ui_automation_tests/step_defs/test_standard_application.py (hunk 4 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/step_defs/test_standard_application.py]: The most recent item (item at the end of the list)

File: ui_automation_tests/step_defs/test_standard_application.py (hunk 5 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/step_defs/test_standard_application.py]: same as above

File: ui_automation_tests/features/submit_standard_application.feature (hunk 3 above)
Comment (BenAndrew-Kainos) [reply ui_automation_tests/features/submit_standard_application.feature]: Added

**Pull request merged**